src/main/scala/com/typesafe/sbt/packager/archetypes/SystemD.scala

```diff
@@ -0,0 +1,24 @@
+package com.typesafe.sbt.packager.archetypes
+
+import sbt._
+import ServerLoader._
+
+// we should put this setting in a more appropriate place
+import com.typesafe.sbt.packager.linux.LinuxPlugin.autoImport.serverLoading
+
+object SystemD extends AutoPlugin {
+
+  object autoImport {
+    // all systemd specific settings/tasks here
+    val systemdExitSuccessStatus = settingKey[Int]("defines the ExitSuccessStatus for systemd")
+  }
+
+  import autoImport._
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/nepomukseiler/sbt-native-packager/file/2460298881/issues/source?bid=2040925&fileBranchId=3254858#l16)